### ISSUE TITLE
[BUG] enum AccessMethods contains dashes where other enums contain underscores

### ISSUE THREAD

**Issue by vmastenb** [TOMP-WG/TOMP-API#470]
## API Version
1.3.0

## Summary
enum AccessMethods (used in object AssetProperties) contains dashes (e.g. TOMP-API or AXA-EKEY-OTP) where other enums use underscores

## Expected Behavior
I would expect the same way of defining enums in the whole project

I am implementing this api in .NET, and in .Net no dashes are allowed in enums. Please change to underscore or CamelCase.

**Comment by edwinvandenbelt** [comment]
Hello Vincent,
when I generate the .net server using the swagger editor, I do get this file:
/*
 * Transport Operator MaaS Provider API
 *
 * An API between MaaS providers and transport operators for booking trips and corresponding assets. <p>The documentation (examples, process flows and sequence diagrams) can be found at <a href=\"https://github.com/TOMP-WG/TOMP-API/\">github</a>.
 *
 * OpenAPI spec version: 1.4.0
 * 
 * Generated by: https://github.com/swagger-api/swagger-codegen.git
 */
using System;
using System.Linq;
using System.IO;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Collections.ObjectModel;
using System.ComponentModel.DataAnnotations;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Models
{ 
        /// <summary>
        /// Gets or Sets assetAccessMethods
        /// </summary>
        [JsonConverter(typeof(Newtonsoft.Json.Converters.StringEnumConverter))]
        public enum AssetAccessMethods
        {
            /// <summary>
            /// Enum DEEPLINKEnum for DEEPLINK
            /// </summary>
            [EnumMember(Value = "DEEPLINK")]
            DEEPLINKEnum = 0,
            /// <summary>
            /// Enum QREnum for QR
            /// </summary>
            [EnumMember(Value = "QR")]
            QREnum = 1,
            /// <summary>
            /// Enum AZTECEnum for AZTEC
            /// </summary>
            [EnumMember(Value = "AZTEC")]
            AZTECEnum = 2,
            /// <summary>
            /// Enum TOMPAPIEnum for TOMP-API
            /// </summary>
            [EnumMember(Value = "TOMP-API")]
            TOMPAPIEnum = 3,
            /// <summary>
            /// Enum AXAEKEYOTPEnum for AXA-EKEY-OTP
            /// </summary>
            [EnumMember(Value = "AXA-EKEY-OTP")]
            AXAEKEYOTPEnum = 4,
            /// <summary>
            /// Enum PHYSICALKEYEnum for PHYSICAL-KEY
            /// </summary>
            [EnumMember(Value = "PHYSICAL-KEY")]
            PHYSICALKEYEnum = 5,
            /// <summary>
            /// Enum BARCODEEnum for BARCODE
            /// </summary>
            [EnumMember(Value = "BARCODE")]
            BARCODEEnum = 6,
            /// <summary>
            /// Enum PDFEnum for PDF
            /// </summary>
            [EnumMember(Value = "PDF")]
            PDFEnum = 7,
            /// <summary>
            /// Enum HTMLEnum for HTML
            /// </summary>
            [EnumMember(Value = "HTML")]
            HTMLEnum = 8,
            /// <summary>
            /// Enum OVCEnum for OVC
            /// </summary>
            [EnumMember(Value = "OVC")]
            OVCEnum = 9,
            /// <summary>
            /// Enum EMVEnum for EMV
            /// </summary>
            [EnumMember(Value = "EMV")]
            EMVEnum = 10,
            /// <summary>
            /// Enum NONEEnum for NONE
            /// </summary>
            [EnumMember(Value = "NONE")]
            NONEEnum = 11        }
}

The generated C# code seems to compile correctly. The EnumMember-annotations are coping with the dash-problem. Does this work for you as well?

**Comment by vmastenb** [comment]
Yes, that works for me as well. So for implementation it is no issue anymore, still it is different than other enums where _underscores_ are used instead of -dashes-

**Comment by edwinvandenbelt** [comment]
Registered for version 2.0, since it's officially a breaking issue

**Comment by edwinvandenbelt** [comment]
MUST

**Comment by edwinvandenbelt** [comment]
https://github.com/TOMP-WG/TOMP-API/commit/7b4960e55396aae8566c831fcf853dc1dfbcf6df
